jobs/ci-kubernetes-kubemark-gce-scale.sh

```diff
@@ -44,7 +44,7 @@ export TEST_CLUSTER_MAX_REQUESTS_INFLIGHT="--max-requests-inflight=1500"
 export LOAD_TEST_THROUGHPUT=25
 # Override defaults to be independent from GCE defaults and set kubemark parameters
 # We need 11 so that we won't hit max-pods limit (set to 100). TODO: do it in a nicer way.
-export NUM_NODES="51"
+export NUM_NODES="59"
```

Comment:
Yes that would be a nice experiment. Do we have such VMs available right now (can't find it in any of the test-infra/jobs)?